defaults/config.js

```diff
@@ -368,4 +368,11 @@ module.exports = {
 	// @default  false
 	//
 	debug: false,
+
+	// Enables logging raw IRC messages into each server window.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	debugRaw: false,
```

Comment:
I hear you and that's a good use case, but current configuration options seem a bit off, and an object might be overkill? Could do an array of strings, that saves a few characters but same stuff in the end...

I don't have anything other than that to suggest, so your idea is probably best:

- `debug: false` means no debug at all
- `debug: true` means all kind of debugs enabled
- `debug: { connection: true, raw: true, ... }` or `debug: ["connection", "raw", ...]` to enable some of the debugging options. To only enable `raw`, `debug: { connection: false, raw: true }` or `debug: { raw: true }` or `debug: ["raw"]`.

That actually seems like a good idea to offer "flags" like that: by default, in prod, most people will have debug options to `false`, but things can get more complex if necessary.
I prefer the array way as it's simpler in the config and reads more like a list of flags.

Then you can have a helper `Helper.config.hasDebug("connection")` that returns `true` if `config.debug === true || (Array.isArray(config.debug) && config.debug.includes("connection"))` or something like that.

`debug: true` doesn't have to be a thing if you want options to be always passed in explicitly, but it's a nice opposite to `debug: false` === all debug flags disabled.